tools/bioimaging/bioimage_inference.xml

```diff
@@ -69,7 +68,7 @@
             <param name="input_image_file" value="input_nucleisegboundarymodel.png"/>
             <param name="input_image_input_size" value="256,256,1,1"/>
             <param name="input_image_input_axes" value="bcyx"/>
-            <output name="output_predicted_image" ftype="tiff">
+            <output name="output_predicted_image" ftype="tiff" file="output_nucleisegboundarymodel.tiff" compare="image_diff">
                 <assert_contents>
                     <has_size size="524846" delta="110" />
                 </assert_contents>
```

Comment:
With the `image_diff` check in place, I think we can remove this?

The `has_size` test is very heuristical, it might even raise a false-negative check failure.

Reply:
sure!